openBCIBoard.js

```diff
@@ -779,8 +917,10 @@ function OpenBCIFactory() {
                         console.log('\tNot calculating impedance for either P and N input.');
                     }
                 }
+                if(pInput) this.impedanceArray[channelNumber - 1].P.raw = this.impedanceTest.impedanceForChannel;
+                if(nInput) this.impedanceArray[channelNumber - 1].N.raw = this.impedanceTest.impedanceForChannel;
                 resolve(channelNumber);
-            }, 250);
+            }, 1000);
```

Comment:
Changed to 400ms, because new Goertzel algorithm will be able to get 5 or so samples out of this time frame, allowing for plenty of time to level off.